examples/single-sign-on/iam-identity-center/sso.tf

```diff
@@ -0,0 +1,262 @@
+  display_name      = "Platform Admin"
+  user_name         = "admin@example.com"
+
+  name {
+    family_name = "Admin"
+    given_name  = "Platform"
+  }
+}
+
+resource "aws_identitystore_user" "user" {
+  identity_store_id = tolist(data.aws_ssoadmin_instances.this.identity_store_ids)[0]
+  display_name      = "Developer User"
+  user_name         = "user@example.com"
+
+  name {
+    family_name = "User"
+    given_name  = "Developer"
+  }
+}
+
+resource "aws_identitystore_group" "operators" {
+  identity_store_id = tolist(data.aws_ssoadmin_instances.this.identity_store_ids)[0]
+  display_name      = "eks-operators"
+  description       = "EKS Operators Cluster Group"
+}
+
+resource "aws_identitystore_group" "developers" {
+  identity_store_id = tolist(data.aws_ssoadmin_instances.this.identity_store_ids)[0]
+  display_name      = "eks-developers"
+  description       = "EKS Developers Cluster Group"
+}
+
+resource "aws_identitystore_group_membership" "operators" {
+  identity_store_id = tolist(data.aws_ssoadmin_instances.this.identity_store_ids)[0]
+  group_id          = aws_identitystore_group.operators.group_id
+  member_id         = aws_identitystore_user.admin.user_id
+}
+
+resource "aws_identitystore_group_membership" "developers" {
+  identity_store_id = tolist(data.aws_ssoadmin_instances.this.identity_store_ids)[0]
+  group_id          = aws_identitystore_group.developers.group_id
+  member_id         = aws_identitystore_user.user.user_id
+}
+
+resource "aws_ssoadmin_account_assignment" "operators" {
+  instance_arn       = tolist(data.aws_ssoadmin_instances.this.arns)[0]
+  permission_set_arn = aws_ssoadmin_permission_set.admin.arn
+
+  principal_id   = aws_identitystore_group.operators.group_id
+  principal_type = "GROUP"
+
+  target_id   = data.aws_caller_identity.current.account_id
+  target_type = "AWS_ACCOUNT"
+}
+
+resource "aws_ssoadmin_account_assignment" "developer" {
+  instance_arn       = tolist(data.aws_ssoadmin_instances.this.arns)[0]
+  permission_set_arn = aws_ssoadmin_permission_set.user.arn
+
+  principal_id   = aws_identitystore_group.developers.group_id
+  principal_type = "GROUP"
+
+  target_id   = data.aws_caller_identity.current.account_id
+  target_type = "AWS_ACCOUNT"
+}
+
+resource "kubernetes_cluster_role_binding_v1" "cluster_admin" {
+  metadata {
+    name = "sso-cluster-admin"
+  }
+  role_ref {
+    api_group = "rbac.authorization.k8s.io"
+    kind      = "ClusterRole"
+    name      = "cluster-admin"
+  }
+  subject {
+    kind      = "Group"
+    name      = "eks-operators"
+    api_group = "rbac.authorization.k8s.io"
+  }
+}
+
+resource "kubernetes_cluster_role_binding_v1" "cluster_viewer" {
+  metadata {
+    name = "sso-cluster-viewer"
+  }
+  role_ref {
+    api_group = "rbac.authorization.k8s.io"
+    kind      = "ClusterRole"
+    name      = "view"
+  }
+  subject {
+    kind      = "Group"
+    name      = "eks-developers"
+    api_group = "rbac.authorization.k8s.io"
+  }
+}
```

Comment:
I am wondering if it makes sense to show how to use teams module here and tie in SSO groups to teams/namespaces?

Reply:
Of course! I can do it.

Reply:
Done! :)